### ISSUE TITLE
error in converting multi output model

### ISSUE THREAD

**Issue by keivanmoazami** [onnx/keras-onnx#677]
I trained center-net for object detection in keras with this repo : https://github.com/xuannianz/keras-CenterNet
model has resenet18 backend and 3 output. when model converting to onnx return error in branching layer.

```
image_input = Input(shape=(image_input_size, image_input_size, 3))
resnet = resnet_models.ResNet18(image_input, include_top=False, freeze_bn=freeze_bn)
C5 = resnet.outputs[-1]
x = Dropout(rate=0.5)(C5)
num_filters = 256
for i in range(3):
    num_filters = num_filters // pow(2, i)
    x = Conv2DTranspose(num_filters, (4, 4), strides=2, use_bias=False, padding='same',
                        kernel_initializer='he_normal',
                        kernel_regularizer=l2(5e-4))(x)
    x = BatchNormalization()(x)
    x = ReLU()(x)

# hm header
y1 = Conv2D(64, 3, padding='same', use_bias=False, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4))(x)
y1 = BatchNormalization()(y1) ### error when converting
y1 = ReLU()(y1)
y1 = Conv2D(num_classes, 1, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4), activation='sigmoid')(y1)

# wh header
y2 = Conv2D(64, 3, padding='same', use_bias=False, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4))(x)
y2 = BatchNormalization()(y2)
y2 = ReLU()(y2)
y2 = Conv2D(2, 1, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4))(y2)

# reg header
y3 = Conv2D(64, 3, padding='same', use_bias=False, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4))(x)
y3 = BatchNormalization()(y3)
y3 = ReLU()(y3)
y3 = Conv2D(2, 1, kernel_initializer='he_normal', kernel_regularizer=l2(5e-4))(y3)

model = Model(inputs=image_input, outputs=[y1, y2, y3])
```
error:
model_2/batch_normalization_3/FusedBatchNormV3:1 is disconnected, check the parsing log for more details.
y1 = BatchNormalization()(y1) under hm header is batch_normalization_3

keras2onnx\topology.py", line 221, in _remove_unused_nodes   
          assert in_ == '' or in_ in input_dict,

**Comment by jiafatom** [comment]
Can you try keras2onnx master and onnxconverter-common master? I just tried your script on master branch, and it seems working for me.

**Comment by judrew** [comment]
I actually have the same issue.
With keras2onnx and onnxconverter-common on version 1.7.x  I had the [onnx.checker issue as in #170](https://github.com/onnx/keras-onnx/issues/170).
After changing to keras2onnx/onnxconverter master versions 1.8.0 it resulted in this issue.

Using tensorflow==2.3.0, Keras==2.4.3

**Comment by jiafatom** [comment]
(1) Can your model run successfully using onnxruntime? If yes, then your inference is fine.
(2) Is the centernet model working with tensorflow==2.3.0? The requirement there seems tensorflow 1.x, see [here](https://github.com/xuannianz/keras-CenterNet/blob/master/requirements.txt#L6)

**Comment by keivanmoazami** [comment]
It work fine with tf 2.4.0 
I rewrite ResNet to use tf.keras instead of keras. but I have new issue in parser.
in method build_layer_output_from_model() and line "graph = model.outputs[0].graph" (line 308 _parser_tf.py)
'KerasTensor' object has no attribute 'graph'

you can download model from below link:
https://drive.google.com/file/d/1QZimubuRW33cYiAODif_SK4GtbmqEeW3/view?usp=sharing

**Comment by jiafatom** [comment]
Currently keras2onnx does not support tf 2.4.0 because tensorflow changes the API so that we cannot extract the underlying graph from the tf.keras model. Is it possible you use tf 2.3? and does tf 2.3 work?

**Comment by keivanmoazami** [comment]
I save keras model as tf and then use tf2onnx and model converted successfully.
